tests/Microsoft.Bot.Builder.Dialogs.Tests/ActivityPromptTests.cs

```diff
@@ -193,8 +193,11 @@ public async Task ActivityPromptResumeDialogShouldReturnDialogEndOfTurn()
             })
             .Send("hello")
             .AssertReply("please send an event.")
+            .AssertReply("please send an event.")
```

Comment:
we want the bot to be repeating itself?

Reply:
In a real-world situation, `ResumeDialogAsync` would be called in response to a dialog on top of this dialog getting popped off of the stack. The extra dialog has been omitted here for simplicity, but you can imagine some dialog happening in between the two instances of "please send an event."

Reply:
@tomlm - I will assume you are satisfied with my answer